doc/whats_new/v1.2.rst

```diff
@@ -318,6 +318,13 @@ Changelog
   :pr:`22710` by :user:`Conroy Trinh <trinhcon>` and
   :pr:`23461` by :user:`Meekail Zain <micky774>`.
 
+- |Fix| add a `"auto"` option to `eps` in :func:`metrics.logloss`.
+  This option will automatically set the `eps` value depending on the data
+  type `y_pred`.
```

Comment:
At this point I would consider this an enhancement which also happens to fix a bug. Wondering what the maintainers think.

```suggestion
- |Enhancement| Adds an `"auto"` option to `eps` in :func:`metrics.logloss`.
  This option will automatically set the `eps` value depending on the data
  type of `y_pred`. In addition, the default value of `eps` is changed from
  `1e-15` to the new `"auto"` option.
```

Reply:
Fair enough.